utils/xscmanager.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	clientconfig "github.com/jfrog/jfrog-client-go/config"
+	"github.com/jfrog/jfrog-client-go/xsc"
+)
+
+func CreateXscServiceManager(serviceDetails *config.ServerDetails) (*xsc.XscServicesManager, error) {
```

Comment:
Please change argument name to serverDetails instead of serviceDetails because server and service mean 2 different things in this context and its best to differentiate between them